src/components/ui/widget-header.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import {StyleSheet, View} from 'react-native';
+
+import {Color} from '@app/colors';
+import {Icon, IconsName, Text} from '@app/components/ui';
+
+type Props = {
+  icon?: IconsName | keyof typeof IconsName;
+  title: string;
+  description?: string;
+  largeIcon?: boolean;
+};
+
+export const WidgetHeader = ({icon, title, description, largeIcon}: Props) => {
```

Comment:
this function can be simplified, because in both situation you have similar markup

Reply:
Please provide an example, because in a closer look you will see, that layout is very different

Reply:
```tsx
const comp = useMemo(() => (
    <View style={styles.column}>
       <View style={styles.row}>
         {!!icon && <Icon name={icon} color={Color.graphicBase1} />}
         <Text t8 style={icon && styles.title}>
           {title}
         </Text>
       </View>
       {!!description && (
         <Text t14 style={styles.description}>
           {description}
         </Text>
       )}
     </View>
), [])

if (largeIcon) {
  return comp
}

return (
<View style={styles.column}>
{comp}
</View>
)
```